### ISSUE TITLE
Support Muisccast App Scrobbling

### ISSUE THREAD

**Issue by schklerg** [kawaiiDango/pano-scrobbler#402]
I use the Yamaha Musiccast App with media directly from my phone. The scrobbler picks up the name of the speaker and my phone as 'server' instead of the artist and song. I would love for musiccast to be parsed better. If there's any data or anything I could provide, I'd be happy to.

Musiccast 5.60 on Android

**Comment by kawaiiDango** [comment]
Please send a bug report from the app, shortly after this happens

**Comment by schklerg** [comment]
Bug report sent. Thank you!

**Comment by kawaiiDango** [comment]
Do both the actual track and the Server - Living Room get scrobbled?

**Comment by schklerg** [comment]
Sometimes. When I was using it while filing the bug report it did, but I have had days when only Server - Living Room shows up. It's inconsistent.

**Comment by kawaiiDango** [comment]
If you are willing to provide additional info since I don't have the hardware,

Here is a build based on [Media Controller Test](https://github.com/googlesamples/android-media-controller): https://transfer.sh/88nuctABWq/mediacontroller-release.apk

That app needs the notification access permission to work.
When you play something with MusicCast, two instances of it should appear on the Media Controller Test app,
one for the actual song and one for "Server".

Click on the "Test" button and take long screenshots of that page for each of the instances.

**Comment by kawaiiDango** [comment]
Please check if v3.18 fixes the issue

**Comment by kawaiiDango** [comment]
Please do the media controller test, otherwise I'm just doing guesswork without the Yamaha hardware to test.

**Comment by schklerg** [comment]
Looks like the apk isn't available any more.

**Comment by kawaiiDango** [comment]
Use this: https://transfer.sh/xDXs01QsmB/mediacontroller-release.apk

**Comment by schklerg** [comment]
![Screenshot_20240221_093633_Media Controller Tester](https://github.com/kawaiiDango/pano-scrobbler/assets/41834624/b79f16c9-e463-487a-ae13-7d0579815d21)
![Screenshot_20240221_093616_Media Controller Tester](https://github.com/kawaiiDango/pano-scrobbler/assets/41834624/8203d4d5-ec59-4bd4-a523-7c3799481c35)

**Comment by kawaiiDango** [comment]
Thank you. Please check if this build of Pano Scrobbler scrobbles correctly from MusicCast

https://transfer.sh/GqM2FhqLoW/pano-scrobbler-release.apk

If it does, I will release it on Play Store

**Comment by schklerg** [comment]
Working flawlessly so far.  Thank you!

**Comment by gdchilla** [comment]
Hi together,

I already wrote an email. I use Musiccast with deezer. Until yesterday it work almost fine with some tricks. After the update the tracks are not found anymore. There isn't only a notification you can click away.

With the older version 3.19 it does get track information but scrobbles artist and album together.Yesterday, neither of these problems have occurred.

There was only the problem that the pano scrobbler did not send the scrobbles to last.fm. My solution was to let the app scrobbler after 30 seconds. Without this setting there was only a "now scrobbling"  shown but no actually scrobbling happens. The problem of artist and album merging did not happen.

**Comment by gdchilla** [comment]
I think I got the problem. I am not using the pano scrobbler for scrobbling the songs I play from my phone to the Musiccast device. I would like to scrobble the songs I play with the Musiccast app directly because the internal deezer plugins doesn't scrobble.

And the big difference is that the informations are shown in different notifications. I made some screenshots using version 3.19. 

Playing from deezer on musiccast
![Screenshot_20240225_214542_Launcher 10](https://github.com/kawaiiDango/pano-scrobbler/assets/161251363/dca900f1-11ab-43f7-b33b-f2442ae4c4e1)

Playing music from the phone
![Screenshot_20240225_214627_Launcher 10](https://github.com/kawaiiDango/pano-scrobbler/assets/161251363/1e013237-5826-466e-9fd7-30176df9a5ac)

**Comment by kawaiiDango** [comment]
Are both the screenshots from 3.19?

**Comment by gdchilla** [comment]
Yes both from 3.19. In this version the app gets the information from the notification from the notification "Plattenspieler" It only merges srtist and album.

**Comment by kawaiiDango** [comment]
Ok, do this: https://github.com/kawaiiDango/pano-scrobbler/issues/402#issuecomment-1937084335 while using Musiccast with deezer

**Comment by gdchilla** [comment]
![Screenshot_20240225_221924_Media Controller Tester](https://github.com/kawaiiDango/pano-scrobbler/assets/161251363/fdd44d6f-350f-42a6-b31e-16e2617a5cc4)

![Screenshot_20240225_221918_Media Controller Tester](https://github.com/kawaiiDango/pano-scrobbler/assets/161251363/dac87f69-4bd6-4069-862e-32a7facf20fd)

**Comment by kawaiiDango** [comment]
Is there only one instance of MusicCast in the list of apps while using deezer? I assume there would be two.

**Comment by gdchilla** [comment]
No there is only one instance. I think this is the difference when you play deezer directly on the Musiccast device. Only if you play music directly from the phone there is a second instance.

**Comment by kawaiiDango** [comment]
Okay, I'll try to fix this issue in the next update.

**Comment by gdchilla** [comment]
Thanks a lot for your work!
